corehq/motech/dhis2/dhis2_config.py

```diff
@@ -46,6 +46,9 @@ class Dhis2FormConfig(DocumentSchema):
     })
     completed_date = DictProperty(required=False)
     datavalue_maps = SchemaListProperty(FormDataValueMap)
+    coordinate = DictProperty(required=False, default={
+        "form_question": "/metadata/location/#text"
```

Comment:
I determined this default value from actually inspecting a case which had the location being sent in the background.